### ISSUE TITLE
When launching with AlwaysOnTop=2 in settings.ini - file menu persistss

### ISSUE THREAD

**Issue by redbone75** [milkdrop2077/MilkDrop3#91]
I'm trying to launch in borderless windowed mode, but when I do by using AlwaysOnTop=2, the top File, Effects, Sprites, Options, Help, and About menus persist. 

![cap](https://github.com/user-attachments/assets/d96a8826-6f2f-4d1e-9090-b7b50db652ef)

If using this option, I would like it to be borderless window mode.

**Comment by milkdrop2077** [comment]
Hello @redbone75 
yes it's an old bug, I will patch it for the next version.
Simply delete your settings.ini file and the borderless mode will work again!

**Comment by redbone75** [comment]
> Hello @redbone75 yes it's an old bug, I will patch it for the next version. Simply delete your settings.ini file and the borderless mode will work again!

I actually believe I found a fix on accident when I looked at the INI after doing fullscreen.

ShowMenu=0


That entry seems to resolve my issue.  No patch needed.  Thank you. I would like to talk to you about possibilities of some other things regarding this in Virtual Pinball.  I think I can open a pretty big door for people to jump all over this.
